packages/experimental/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,81 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { tabsName, TabsType, TabsProps, TabsContextData } from './Tabs.types';
+import { Text } from '@fluentui-react-native/experimental-text';
+import { stylingSettings } from './Tabs.styling';
+import { compose, mergeProps, withSlots, UseSlots } from '@fluentui-react-native/framework';
+import { useTabs } from './useTabs';
+import { FocusZone } from '@fluentui-react-native/focus-zone';
+
+export const TabsContext = React.createContext<TabsContextData>({
+  selectedKey: null,
+  onTabsClick: (/* key: string */) => {
+    return;
+  },
+  getTabId: (/* key:string, index: number*/) => {
+    return null;
+  },
+  updateSelectedTabsItemRef: (/* ref: React.RefObject<any>*/) => {
+    return;
+  },
+  tabsItemKeys: [],
+  views: null,
+});
+
+export const Tabs = compose<TabsType>({
+  displayName: tabsName,
+  ...stylingSettings,
+  slots: {
+    root: View,
+    label: Text,
+    container: FocusZone,
+    stack: View,
+    tabPanel: View,
+  },
+  render: (userProps: TabsProps, useSlots: UseSlots<TabsType>) => {
+    const tabs = useTabs(userProps);
+    // Grab the styled slots.
+    const Slots = useSlots(userProps, layer => tabs.state[layer] || userProps[layer]);
+    // Return the handler to finish render.
+    return (final: TabsProps, ...children: React.ReactNode[]) => {
+      const { label, defaultTabbableElement, isCircularNavigation, ...mergedProps } = mergeProps(tabs.props, final);
+
```

Comment:
Add `if(!tabs.state) return`

Reply:
Since we're using it below, let's make sure we don't crash if we're in a weird state